week-2/implement/cases.js

```diff
@@ -15,3 +15,12 @@
 
 // Come up with a clear, simple name for the function
 // Use the string documentation to help you plan your solution
+
+
+function snake_case (str){
+    return str.replaceAll(/ /g,'_').toUpperCase();
```

Comment:
Ah I see you've had an introduction to RegEx, Areeb! It's an interesting technology to use, and can get incredibly complex (if you want an example, look up the RegEx required to check an email address!)

Because RegEx is so complex, I would recommend keeping this as simple as possible by using `.replaceAll(' ', '_')` to just look out for space characters.